compiler/noirc_frontend/src/tests.rs

```diff
@@ -51,11 +52,19 @@ mod test {
         src: &str,
     ) -> (ParsedModule, Context, Vec<(CompilationError, FileId)>) {
         let root = std::path::Path::new("/");
-        let fm = FileManager::new(root);
+        let mut fm = FileManager::new(root);
+        let entrypoint = Path::new("main.nr");
+        let root_file_id = fm.add_file_with_source(entrypoint, src.to_string()).unwrap();
+        let path_to_std_lib_file = Path::new("std").join("lib.nr");
+        let std_file_id =
+            fm.add_file_with_source_canonical_path(&path_to_std_lib_file, "".to_string()).unwrap();
+
         let mut context = Context::new(fm, Default::default());
         context.def_interner.populate_dummy_operator_traits();
-        let root_file_id = FileId::dummy();
+
         let root_crate_id = context.crate_graph.add_crate_root(root_file_id);
+        context.crate_graph.add_stdlib(std_file_id);
+
```

Comment:
I've modified this as we now cannot compile without a stdlib crate. Alternatively we can modify `context.stdlib_crate_id()` to return an `Option<CrateId>` which would avoid this need.

Reply:
I realised that we can call `crate_id.is_stdlib()` rather than doing an explicit equality check so we can avoid this.